### PR TITLE
Mr-OurTable testing - Reduced runtime and maintained testing accuracy

### LINKED ISSUE #13: Improve performance of tests for OurTable
The tests for OurTable take a very long time to run, possibly because someone in the past was a little overzealous about writing the tests.

The challenge in this issue: see if you can come up with a set of tests that still gets you 100% line, branch and mutation coverage, but runs a whole lot faster, and is simpler and shorter than the current test suite.

### PR DESCRIPTION
**Pull request by mayarosenbaum**
## Overview

In this PR, I rewrote the OurTable.test.js code to more efficiently test the ourTable.js and produce 100% mutation,line,and branch coverage. I sped up the testing by about 9 minutes.

## Screenshots (Optional)

Below is the original mutation testing runtime:
OriginalMutationTime
Below is the new mutation testing runtime, still with 100%.
[
NewMutationTime
](url)
Below is the new code's 100% branch/line coverage.
Coverage

## Feedback Request (Optional)

Please give suggestions on the following:
- I stuck with 100 rows for most of the testing. I'm not sure this is necessary, what do you think?

## Future Possibilities (Optional)

Some possible points could be:
- This is important in possibly testing new features for OurTable. The previous test suite ran really slowly so hopefully this will make testing more efficient.

## Validation (Optional)

1. Download this branch.
2. Run mutation testing for OurTable.js to see runtime/accuracy.
3. Run coverage testing for OurTable.js to see branch/line coverage.

## Tests

- [x] Frontend Unit tests (`npm test`) pass
- [x] Frontend Test coverage (`npm run coverage`) 100%
- [x] Frontend Mutation tests (`npx stryker run`) 100% 

## Linked Issues

Closes #13

### CODE REVIEW

**Review by yixuan-wong — DISMISSED** [review]
LGTM! I ran and tested locally and I achieved the same results.
image
image

**Review by yixuan-wong — APPROVED** [review]
Still LGTM!
image
image

**Review by pconrad — CHANGES_REQUESTED** [review]
Please remove package-lock.json and package.json from this PR, since package.json is not changing

File: frontend/package.json [hunk 1]
```diff
@@ -89,4 +89,4 @@
       "lcov"
```
Comment (pconrad) [inline frontend/package.json:89]: please remove package.json and package-lock.json from the PR

**Review by pconrad — CHANGES_REQUESTED** [review]

**Comment by pconrad** [comment]
replaced by #71

**Pull request closed without merging**